app/components/homepage-components/Hero.tsx

```diff
@@ -28,17 +30,23 @@ const Hero = () => {
           to excellence.
         </p>
         <div className="mt-9 flex flex-col md:flex-row md:justify-between md:gap-4">
-          <button className="w-full rounded-full bg-primaryLight py-3.5 text-sm font-semibold text-white900 dark:bg-primaryDark md:w-[15rem] md:py-5 md:text-lg">
+          <Button
```

Comment:
All of the changes must be made on a new branch in a separate Pull Request. Please.